### PR TITLE
Add sleekxmpp and ligo-lvalert

### PR DESCRIPTION
**Pull request by duncanmmacleod**
This PR adds two new recipes:

- [`sleekxmpp`](https://pypi.org/project/sleekxmpp/) - **I am not a developer or maintainer on this project**
- [`ligo-lvalert`](https://pypi.org/project/ligo-lvalert/) - used for event notifications in GW astrophysics

cc: @aepace, @tprestegard as co-maintainers of `ligo-lvalert`.

### CODE REVIEW

File: recipes/sleekxmpp/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+    - sleekxmpp.features.feature_bind
+    - sleekxmpp.features.feature_session
+    - sleekxmpp.features.feature_rosterver
+    - sleekxmpp.features.feature_preapproval
+    - sleekxmpp.thirdparty
+
+about:
+  home: https://github.com/fritzy/SleekXMPP
+  license: MIT
+  license_family: MIT
```
Comment (marcelotrevisani) [inline recipes/sleekxmpp/meta.yaml]: Please, add the ``license_file``

**Comment by duncanmmacleod** [comment]
@conda-forge/staged-recipes, I think this is ready for review, thanks.

File: recipes/ligo-lvalert/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+  summary: LVAlert Client Tools
+  description: |
+    LVAlert is an XMPP-based alert system. This package provides client
+    tools for interacting with the LVAlert jabber server.
+
+extra:
+  recipe-maintainers:
+    - duncanmmacleod
+    - tprestegard
+    - aepace
```
Comment (scopatz) [inline recipes/ligo-lvalert/meta.yaml]: @aepace @tprestegard - are you OK with being maintainers here?

**Comment by duncanmmacleod** [comment]
Thanks @scopatz!

**Pull request merged**